content/en/tracing/llm_observability/sdk.md

```diff
@@ -59,6 +59,49 @@ DD_LLMOBS_APP_NAME=<YOUR_ML_APP_NAME> ddtrace-run <YOUR_APP_STARTUP_COMMAND>
 : optional - _integer or string_ - **default**: `false`
 <br />Only required if you are not a Datadog APM customer, in which case this should be set to `1` or `true`.
 
+#### In-code setup
+
+Enable LLM Observability programatically through the `LLMOBs.enable()` function instead of running with the `ddtrace-run` command. **Note**: This setup method will not work if used with `ddtrace-run`.
```

Comment:
Can we change this to
"Do not use this setup method with `ddtrace-run`"

Agree that we don't need to say all the details why, but a firm instruction is better than using the term "not work" since then we need to explain what "not work" means